app/Followers/page.tsx

```diff
@@ -0,0 +1,65 @@
+import { getDocs, collection } from "firebase/firestore";
+
+import FeedPage from "@/components/feed";
+import FilterSavedFeedButton from "@/components/feed/FilterSavedFeedButton";
+import PageTitle from "@/components/ui/PageTitle";
+import { getFeedItems } from "@/lib/feed";
+import firestore from "@/lib/firestore";
+import isUserFollowing from "@/lib/followers/isUserFollowing";
+import { getCurrentUser } from "@/lib/getUser";
+import { getSavedFeedItemsList } from "@/lib/savedFeedItems";
+
+type FollowingUser = {
+  id: string;
+};
+
+const Feed = async () => {
+  const feedItems = await getFeedItems();
+  const currentUser = await getCurrentUser();
+  // TODO : get following user ids for current user
+  async function getFollowingUserIds() {
+    const followingCol = collection(
```

Comment:
There's no such thing as a following collection. You should get all users, get their IDs, and then check if they're followed using Spotify's API.